### ISSUE TITLE
3rd party viewing of Destiny Collections

### ISSUE THREAD

**Issue by jamesdowdall** [fafranco82/swdestinydb#37]
The collections functionality provided by Destiny is very useful for tracking your own collection, however it would be very useful for trading purposes if I could share my collection with my friends/local player base. Obviously this should be opt in, but it would save a lot of time when asking who has a given card. Further enhancements could include creating user groups, where players can all share their cardsets.

Note this is functionally different to https://github.com/fafranco82/swdestinydb/issues/20 I'm talking about letting other people see your own collection

**Comment by fafranco82** [comment]
Yes, it will be a great idea. Indeed, you can now share your deck without publishem them via an option... I can include another one to let your collection public (without modify permission, of course).

**Comment by skitau** [comment]
Was looking to do this also, a great feature to assist trades.
Would also be great to sort or filter on cards in my collection that are greater than 2.

**Comment by jgoldshlag** [comment]
Another vote for this, would be super helpful

**Comment by jonsoini** [comment]
Thanks for your work on this. Being able to share collections or download a csv/xls would be awesome.

**Comment by tillmannen** [comment]
Would it be possible to add a way to get your collection with the API? That way we could build 3:d party apps for trade and/or for managing your collection.
